.github/workflows/e2e-windows.yml

```diff
@@ -17,8 +13,17 @@ on:
         default: 'all'
 jobs:
   test:
+
     runs-on: windows-latest
 
+    env:
```

Comment:
this setup seems like the one we can extract and reuse also

Reply:
I totally support this, I also tried and failed. Can we move it to a seprate Ticket, I am happy to work on that after the release